openwisp_firmware_upgrader/admin.py

```diff
@@ -64,6 +65,10 @@ class BuildAdmin(BaseVersionAdmin):
     inlines = [FirmwareImageInline]
     actions = ['upgrade_selected']
     multitenant_parent = 'category'
+    change_form_template = 'admin/firmware_build/change_form.html'
+
+    class Media:
+        css = {'all': (static('firmware_build/css/upgrade_build.css'),)}
```

Comment:
change this file so that it's available at this path: `admin/css/firmware-upgrader.css` (for consistency with other modules)